lighthouse-core/audits/resource-summary.js

```diff
@@ -76,14 +76,14 @@ class ResourceSummary extends Audit {
         resourceType: type,
         label: strMappings[type],
         requestCount: summary[type].count,
-        size: summary[type].size,
+        transferSize: summary[type].transferSize,
```

Comment:
note to self: lighthouse CI will need to be updated to account for this as part of https://github.com/GoogleChrome/lighthouse-ci/issues/119